### PR TITLE
rename "some other race" to "unrepresented race"

### LINKED ISSUE #1236: Use more inclusive language for category labels
Currently `Some other race` is used per CDC terminology; many find this to be a dismissive term and there is room for improvement. We have received feedback from HETF

Additionally, the term `Other` for Sex seems IMHO even worse. Currently the discussion has revolved around our racial categories but we should strongly consider this label as well. Perhaps using the same approach would work: `An unrepresented sex`

### PR DESCRIPTION
**Pull request by joshzarrabi**
## Description

Rename the "OTHER_STANDARD" race from "Some other race" to "Unrepresented race"

## Motivation and Context

[Closes #1236]

## Screenshots (if appropriate):

Screen Shot 2022-01-31 at 2 19 37 PM

## Types of changes

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by joshzarrabi** [comment]
Options for multi category with screenshots 

* Two or more races & Unrepresented ra
Screen Shot 2022-02-07 at 2 47 33 PM
ce (Non-Hispanic) 

* Two or more races or Unrepresented race (Non-Hispanic)
Screen Shot 2022-02-07 at 2 49 48 PM

* Two or more races & 'Unrepresented race' (Non-Hispanic)
Screen Shot 2022-02-07 at 2 49 29 PM

File: frontend/src/pages/DataCatalog/MethodologyTab.tsx [hunk 1]
```diff
@@ -428,19 +428,24 @@ function MethodologyTab() {
...
-                    in the dataset.
+                    <b>Unrepresented race (Non-Hispanic)</b>: A single race not
+                    tabulated by the CDC, not of Hispanic/Latino ethnicity.
+                    Individuals not identifying as one of the distinct races
+                    listed in the source data, or multiracial individuals, are
+                    grouped together as “Some other race”. This is a problem as
+                    it obscures racial identity for many individuals. In our
+                    effort to take transformative action towards achieving
+                    health equity the Satcher Health Leadership Institute has
+                    decided to rename this category to highlight it as a Health
```
Comment (benhammondmusic) [inline frontend/src/pages/DataCatalog/MethodologyTab.tsx:439]: should "Health Equity Issue" be lowercased? I know Mahia has pushed us away from title casing and to use normal sentence casing

Comment (joshzarrabi) [reply frontend/src/pages/DataCatalog/MethodologyTab.tsx:439]: Yeah good catch

**Review by benhammondmusic — APPROVED** [review]

**Review by benhammondmusic — APPROVED** [review]
looks good. glad we are making this change

**Pull request merged**